libraries/ts-command-line/src/providers/ScopedCommandLineAction.ts

```diff
@@ -180,23 +179,9 @@ export abstract class ScopedCommandLineAction extends CommandLineAction {
       throw new Error('Parameters must be defined before execution.');
     }
 
-    // The '--' argument is required to separate the action parameters from the scoped parameters,
-    // so it needs to be trimmed. If remainder values are provided but no '--' is found, then throw.
-    const scopedArgs: string[] = [];
-    if (this.remainder.values.length) {
-      if (this.remainder.values[0] !== '--') {
-        throw new CommandLineParserExitError(
-          // argparse sets exit code 2 for invalid arguments
-          2,
-          // model the message off of the built-in "unrecognized arguments" message
-          `${this.renderUsageText()}\n${this._unscopedParserOptions.toolFilename} ${this.actionName}: ` +
-            `error: Unrecognized arguments: ${this.remainder.values[0]}.\n`
-        );
-      }
-      for (const scopedArg of this.remainder.values.slice(1)) {
-        scopedArgs.push(scopedArg);
-      }
-    }
```

Comment:
Use exactly this logic as how to build the remainder, this is the desired behavior.